### PR TITLE
Feature/product pages

### PR DESCRIPTION
**Pull request by DanHowardDado**
Commented out the previous collections and card item tasks due to changes in the content models for this product page task.
Previous tasks will be updated and new PRs created.

This PR addresses the product page, it basically grabs contentful and shopify product data, creates a separate array of objects for the two and then populates the product selector options with the two arrays

Will implement the add to cart/ subscribe button in the next ticket

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -1,58 +1,110 @@
...
+//}
+//allContentfulShopifyProduct {
+//    edges {
+//        node {
+//            title
+//            subtitle
+//            variantId
+//       }
+//    }
+//}
```
Comment (martink-rsa) [inline gatsby-node.js:22]: Remove commented code before merging

File: gatsby-node.js [hunk 2]
```diff
@@ -1,58 +1,110 @@
...
+
+    // createPage({
+    //     path: `/product/poc/`,
+    //     component: path.resolve(`./src/components/CardProductsSection/index.js`),
+    //     context: {
+    //         // Data passed to context is available
+    //         // in page queries as GraphQL variables.
+    //         productIds: ids
+    //     },
+    // })
```
Comment (martink-rsa) [inline gatsby-node.js:91]: Remove commented code before merging

File: src/components/CategoryPage/index.js [hunk 3]
```diff
@@ -6,18 +6,18 @@ import { Title } from './styles'
...
-  console.log(shopifyArr)
+  // const updateShopifyData = () => {
+  //   shopifyArr.map(function ({ node }) {
+  //     var result = contentfulArr.filter(contentful => contentful.variantId == node.variants[0].shopifyId); //might need an in here insead of ==
+  //     if (result.length > 0) { node.title = result[0].title; node.subtitle = result[0].subtitle; }
+  //     return node
+  //   })
+  // };
+  // updateShopifyData();
+  // console.log(shopifyArr)
```
Comment (martink-rsa) [inline src/components/CategoryPage/index.js:20]: Remove commented code before merging

File: src/components/CategoryPage/index.js [hunk 4]
```diff
@@ -28,10 +28,10 @@ const CategoryPage = ({ data, pageContext }) => {
     }).format(parseFloat(price ? price : 0))
   return (
     <Container>
-      <Title>
+      {/* <Title>
         {pageContext.collectionName}
-      </Title>
-      <FlexGrid>
+      </Title> */}
+      {/* <FlexGrid>
```
Comment (martink-rsa) [inline src/components/CategoryPage/index.js:34]: Remove commented code before merging

File: src/components/CategoryPage/index.js [hunk 5]
```diff
@@ -47,41 +47,41 @@ const CategoryPage = ({ data, pageContext }) => {
...
+//             }
+//             variants {
+//               price
+//               shopifyId
+//             }
+//           }
+//         }
+//       }
+//     }
+//   `
```
Comment (martink-rsa) [inline src/components/CategoryPage/index.js:85]: Remove commented code before merging.

File: src/components/ProductSelector/index.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+  // updateShopifyData();
+  // console.log(shopifyArr)
+
+  // const getPrice = price =>
+  //   Intl.NumberFormat(undefined, {
+  //     currency: 'USD',
+  //     currencyDisplay: 'narrowSymbol',
+  //     minimumFractionDigits: 2,
+  //     style: 'currency',
+  //   }).format(parseFloat(price ? price : 0))
```
Comment (martink-rsa) [inline src/components/ProductSelector/index.js:27]: Remove comments before merging.

File: src/components/ProductSelector/index.js [hunk 7]
```diff
@@ -0,0 +1,86 @@
+import React from 'react'
```
Comment (martink-rsa) [inline src/components/ProductSelector/index.js:1]: Is this component needed? It doesn't seem to have much purpose with the code commented out as it only has a single container, but no children.

**Comment by DanHowardDado** [comment]
Thanks for reviewing this PR @martink-rsa

File: src/components/ProductSelector/index.js (hunk 7 above)
Comment (DanHowardDado) [reply src/components/ProductSelector/index.js:1]: Nope, was developing a story for this component but will do so in another branch and remove this from the PR thanks

**Pull request merged**